operator/attestor/notifier.go

```diff
@@ -28,7 +28,7 @@ func (notifier *Notifier) Subscribe(rollupId uint32) (<-chan consumer.BlockData,
 		notifier.rollupIdsToSubscribers[rollupId] = list.New()
 	}
 
-	notifierC := make(chan consumer.BlockData, 100)
```

Comment:
Shall be resolved with this one
https://github.com/NethermindEth/near-sffl/pull/131/files